src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java

```diff
@@ -89,6 +78,14 @@ public void init() {
             trigger.isIgnoreSsl());
   }
 
+  // For unit tests only
```

Comment:
not quite true, as it's also called from the other constructor

Reply:
The only outside use is by unit tests. I'm not changing that part.

Reply:
I know, it's minor. However I think "visible for (unit) tests" would be a better comment

Reply:
OK, I'll change it. I'm actually thinking of creating StashApiClient in StashBuildTrigger, it which case the comment would not apply, but that would be a bigger change.

Reply:
Thanks. Though just to clarify: I wanted to just point it out, I don't think it's important enough that it has to be changed